packages/api/src/routes/internal.ts

```diff
@@ -121,4 +128,50 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/upload-doc
+ *
+ * Upload doc for a patient
+ *
+ * @param req.query.cxId - The customer/account's ID.
+ * @param req.query.patientId - The patient ID.
+ * @param req.file - The file to be stored.
+ * @param req.body.metadata - The metadata for the file.
+
+ * @return 200 Indicating the file was successfully uploaded.
+ */
+router.post(
+  "/upload-doc",
+  upload.single("file"),
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const patientId = getFromQueryOrFail("patientId", req);
+    const file = req.file;
+
+    if (!file) {
+      throw new BadRequestError("File must be provided");
+    }
+
+    await s3client
+      .upload({
+        Bucket: bucketName,
+        Key: file.originalname,
```

Comment:
I suggest we use the same folder/filename structure we have for prod - [link](https://github.com/metriport/metriport/blob/4502bdcd3ad4398061b8f13e6710da98a4fc7990/packages/api/src/shared/external.ts#L5)